torch/_tensor_docs.py

```diff
@@ -3202,10 +3238,39 @@ def callable(a, b) -> number
                r"""
 sparse_dim() -> int
 
-If :attr:`self` is a sparse COO tensor (i.e., with ``torch.sparse_coo`` layout),
+If :attr:`self` is a :ref:`sparse tensor <sparse-docs>`,
 this returns the number of sparse dimensions. Otherwise, this throws an error.
 
-See also :meth:`Tensor.dense_dim`.
+See also :meth:`Tensor.dense_dim` and :ref:`hybrid tensors <sparse-hybrid-docs>`.
+""")
+
+add_docstr_all('sparse_resize_',
+               r"""
+sparse_resize_(size, sparse_dim, dense_dim) -> Tensor
+
+Resizes :attr:`self` :ref:`sparse tensor <sparse-docs>` to the
+desired (non-shrinking) size and the number of sparse and dense
+dimensions.
+
+Args:
+    size (torch.Size): the desired size.
+    sparse_dim (int): the number of desired sparse dimensions
+    dense_dim (int): the number of desired dense dimensions
+
+If the number of specified elements of :attr:`self` tensor is 0, one
```

Comment:
A paragraph like this elaborating on the "non-shrinking" behavior is good but should probably be a note or warning immediately after the one-sentence description.

"If the number of specified elements in self is zero, then size, sparse_dim, and dense_dim can be any size such that 
``len(size) == sparse_dim + dense_dim``. If self specifies one or more elements, however, then each dimension in size must be weakly greater than the corresponding dimension of self, sparse_dim must equal the number of sparse dimensions in self, and dense_dim must equal the number of dense dimensions in self." 

This function seems very awkward when self specifies on or more elements, doesn't it? Because it requires populating args, sparse_dim and dense_dim, that are inferrable from self and contain no additional information, but if wrong will throw a runtime error.

Reply:
I like your note suggestion better. Updated.